vm/src/frame.rs

```diff
@@ -203,6 +203,7 @@ enum BlockType {
         end: bytecode::Label,
         context_manager: PyObjectRef,
     },
+    ExceptHandler {},
```

Comment:
Is the syntax without the `{}` valid? I would assume so, like this:
```rust
ExceptHandler,
```

Reply:
Done